### ISSUE TITLE
Optimize Training Configuration - Sound

### ISSUE THREAD

**Issue by julca2706** [AIML21/main#32]
Modify training parameters like learning rate and batch size to accommodate the new sensor input.
Use Unity’s Profiler to measure the impact of adding sound observations on training performance.
